src/main/actions/makeApiClientRequest.js

```diff
@@ -40,6 +40,7 @@ const makeApiClientRequest = async ({ apiRequest }) => {
       data: body,
       responseType: "arraybuffer",
       withCredentials: false,
+      validateStatus: false,
```

Comment:
according to the [docs](https://axios-http.com/docs/handling_errors) and a cursory reading of the tests inside axios around this config, this is always expected to be a function that should determine which status codes to fail for. 

being `undefined | null` has a documented behaviour to use the default config, which is:
```
validateStatus: (status) => {
  return status >= 200 && status < 300; // default
},
```

couldn't determine from their code what happens when explicitly mentioned as `false` here.

if it's working, great! but I would still suggest to rewrite this as:

```suggestion
      validateStatus: () => {
          return true; // never throw error for any status code
        },
```